rosdep/python.yaml

```
@@ -1626,6 +1626,10 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python-flask-restplus-pip:
```

```suggestion
python3-flask-restplus-pip:
```

Per our [contribution guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1) this should be changed to `python3-flask-restplus-pip`.